projects/ng-swiper/src/lib/swiper/item/item.directive.ts

```
@@ -0,0 +1,22 @@
+import { SwiperOptions } from 'swiper/types';
+
+@Directive({
+  selector: '[ngSwiperItem]',
+  host: {
+    '[id]': 'id',
+    '[class]': 'options.slideClass || "swiper-slide"',
+    'aria-roledescription': 'slide'
+  }
+})
+export class SwiperItemDirective {
+  static nextId: number = 0;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)